test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -4378,6 +4465,10 @@ static int traffic_mngr_check_query(void)
 	if ((tm_capabilities.tm_queue_query_flags & query_flags) != query_flags)
 		return ODP_TEST_INACTIVE;
 
+	/* This test needs 64Kbps */
+	if (tm_shaper_min_rate > 64 * 1000)
```

Comment:
The check below the comment does not check that 64 Kbps (please use space before the Kbps unit) is actually supported, since max rate could be lower than that. I first thought that it is a bug, but it seems to me that the called test function does not really care if the rate is smaller. If so, then the comment should be fixed to match the code. It could be something like this: "This test needs a rate of at most 64 Kbps".

Reply:
Currently I'm checking if min rate and max rate allows 64Kbps to be configured. Does the comment sound correct now ?

Reply:
The comment does not match the code. If the comment says "needs a rate of at most 64 kbps", then a smaller rate should be ok too. But it isn't, so the earlier comment was more correct. Either remove the comment or change it to "this test uses 64 Kbps rate and a 1000 bit burst size".